bin/run-ted.sh

```diff
@@ -8,7 +8,7 @@ fi;
 
 python -u DeepSpeech.py \
   --importer ted \
-  --dataset_path /data/LIUM/ \
+  --dataset_path data/LIUM/ \
```

Comment:
Looking at line 43 of DeepSpeech.py[[1](https://github.com/mozilla/DeepSpeech/blob/master/DeepSpeech.py#L43)]
```python
...
tf.app.flags.DEFINE_string ('dataset_path', '', 'data set path for the importer - defaults to ./data/<importer>')
...
```
I think it might be better to simply remove the line
```bash
   --dataset_path /data/LIUM/ \
```